MAUI/Cards/getting-started.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: Getting started with MAUI Card control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Card (SfCalendar) control and its basic features.
+platform: maui
+control: SfCard
+documentation: ug
+---
+
+# Getting started with .NET MAUI Card (SfCard)
+This section details the process of integrating the Card control and focuses solely on the fundamental features required for initiating your exploration of Syncfusion Card.
+
+
+## Creating an application using the .NET MAUI Card
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfCard` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Card](), and then install it.
```

Comment:
2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfCard` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Card](), and then install.